docs/api-docs/channels/channel-webhooks.md

```diff
@@ -25,58 +24,65 @@ Updates to the following fields trigger a `store/channel/updated` event:
 * `is_enabled` (to be deprecated)
 * `config_meta`
 
+## Channel-specific webhook events
+
+The following webhook events fire in response to actions that affect one specific channel on a store:
+
+| Name                                                   | Description                                               | Endpoint |
+|:-------------------------------------------------------|:----------------------------------------------------------|:---------|
+| store/channel/{channel_id}/product/assigned            | Fires when a product is assigned to the specified channel | [Create product channel assignments](/api-reference/store-management/catalog/products-channel-assignments/createproductchannelassignments) |
+| store/channel/{channel_id}/product/unassigned          | Fires when a product is removed from the specified channel | [Delete product channel assignments](/api-reference/store-management/catalog/products-channel-assignments/deleteproductchannelassignments) |
+| store/channel/{channel_id}/category/product/assigned   | Fires when a product is assigned to a category in the specified channel's category tree | [Create product category assignments](/api-reference/store-management/catalog/products-category-assignments/deleteproductscategoryassignments) |
+| store/channel/{channel_id}/category/product/unassigned | Fires when a product is removed from a category in the specified channel's category tree | [Delete product category assignments](/api-reference/store-management/catalog/products-category-assignments/createproductscategoryassignments) |
+| store/channel/{channel_id}/settings/profile/updated    | Fires when the store profile settings that apply to the specified channel are updated | [Update store profile settings](/api-reference/store-management/settings/store-profile/putstoreprofilesettings) |

```

Comment:
@kristinapototska 
Does the `store/channel/{channel_id}/settings/profile/updated` webhook fire only when channel-specific overrides are updated, or also when global store settings are updated for which channel-specific overrides do not exist?

cc @traci707 @markcmurphy